packages/core/src/html/includePanelProcessor.js

```diff
@@ -58,16 +57,26 @@ function _getBoilerplateFilePath(node, filePath, config) {
  * Retrieves several flags and file paths from the src attribute specified in the element.
  */
 function _getSrcFlagsAndFilePaths(element, config) {
-  const isUrl = urlUtil.isUrl(element.attribs.src);
+  let isUrl = urlUtil.isUrl(element.attribs.src);
 
   // We do this even if the src is not a url to get the hash, if any
-  const includeSrc = url.parse(element.attribs.src);
+  let newURL = `${config.baseUrl ? config.baseUrl : 'http://127.0.0.1:8080'}${element.attribs.src}`;
```

Comment:
I don't think you should prepend the `src` with `http://127.0.0.1:8080`. Is there a reason for doing this?

Reply:
The new API, `WHATWG URL API`, that is in replacement of the deprecated one will throw an error `message=Invalid URL, stack=TypeError [ERR_INVALID_URL]: Invalid URL` when there is no `protocol` (e.g. http/https) and `host` (e.g. sub.example.com:8080) passed into the `URL` constructor. That was why I prepended the `src` with `http://127.0.0.1:8080`. 

This might be helpful [here](https://nodejs.org/api/url.html#new-urlinput-base)! On the usage of `URL` constructor.

![image](https://user-images.githubusercontent.com/46486515/184464920-7a48649b-5352-40ae-8922-7466e5169182.png)